web/src/components/domains/task/TaskList/TaskList.jsx

```diff
@@ -1,14 +1,43 @@
 import styles from './TaskList.module.css';
+import TaskItem from '../TaskItem/TaskItem';
+import { PropTypes } from 'prop-types';
 
 /*
 Please create the <TaskList /> component following the design from the Figma file.
 Please make sure to add styles using CSS Modules.
 Create a taskItems array and return a list of <TaskItem /> components.
 */
 
-export function TaskList() {
+function TaskList(props) {
   // create some task items here and return one task list for each item you have
-  const taskItems = [];
-
-  return <div className={styles.listWrapper}></div>;
+  return (
+    <div className={styles.listWrapper}>
+      {props.TaskListInfo.map((listItem, index) => {
+        return (
+          <TaskItem
+            key={index}
```

Comment:
I would recommend that you **not use only the index as key when you are rendering a list** on React because if in the future you will edit that list, you can create confusing errors.

ℹ️ You can find deeply explanation on the Rendering List article of Rect documentation

✨ Tip: In an ideal scenario, in the data of your item, you will have a unique id/sku/... that you can use as a key. If this is not your case, you can concatenate the index with a probably unique value of your data (like the name), like the below example:
```suggestion
            key={`${listItem.ProjectTask}-${index}`}
```

Reply:
Thank you for the feedback and insight on why indexing  might not ALWAYS be the best way.